components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -19,100 +28,117 @@ public function __construct() {
 			add_action( 'wp', array( $this, 'set_frontier_style_script' ) );
 		}
 
+		// Setup initial hooks.
 		add_action( 'template_redirect', array( $this, 'hook_content' ) );
+		// Setup hooks after each new post in the loop.
+		add_action( 'the_post', array( $this, 'hook_content' ) );
 
+		// Setup comment hooks.
+		add_action( 'template_redirect', array( $this, 'maybe_hook_content' ) );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 1c26da7dc4fc1e433ee20471d8841d67</sub>